prow/plugins/label/label.go

```diff
@@ -119,6 +119,15 @@ func handle(gc githubClient, log *logrus.Entry, e *github.GenericCommentEvent) e
 			continue
 		}
 
+		for _, label := range labels {
+			labelPrefix := strings.Split(labelToAdd, "/")
+			if strings.HasPrefix(label.Name, labelPrefix[0]) {
+				if err := gc.RemoveLabel(org, repo, e.Number, label.Name); err != nil {
```

Comment:
please only do this for queue and priority.

Reply:
done  PTAL @ddysher